src/styles/index.scss

```diff
@@ -1,3 +1,3 @@
-body {
-  margin: 0;
-}
+@import "./normalize.scss";
+@import "./card.scss";
+@import "./stars.scss";
```

Comment:
Add a space between the `@import` rule and the file path for better readability.